modular_skyrat/modules/importedvendors/code/vendor_food.dm

```diff
@@ -0,0 +1,276 @@
+	icon_state = "foodtray_chickiesandwich"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/capsaicin = 10)
+	tastes = list("bread" = 2, "chicken?" = 2, "overwhelming spice" = 2, "laziness" = 1)
+	foodtypes = MEAT | GRAIN | DAIRY
+
+/obj/item/food/vendor_tray_meal/ramen
+	name = "\improper メインコース: Homestyle Noodles"
+	desc = "A brick of the finest factory made ramen, with a small amount of rehydrated vegetables and herbs floating around."
+	icon_state = "foodtray_noodle"
+	tastes = list("cheap noodles" = 2, "laziness" = 1)
+	foodtypes = GRAIN | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/sushi
+	name = "\improper メインコース: Fresh Carp Rolls"
+	desc = "A pair of sushi rolls, the appearance of which would suggest that the label is lying to you."
+	icon_state = "foodtray_gasstationsushi"
+	tastes = list("imitation space carp" = 2, "stale rice" = 2, "laziness" = 1)
+	foodtypes = GRAIN | SEAFOOD
+
+/obj/item/food/vendor_tray_meal/beef_rice
+	name = "\improper メインコース: Beef and Fried Rice"
+	desc = "A few slices of seemingly grilled beef, paired with far too much rice for how much meat there is."
+	icon_state = "foodtray_beefnrice"
+	tastes = list("cheap beef" = 1, "rice" = 3, "laziness" = 1)
+	foodtypes = GRAIN | MEAT
+
+/obj/item/food/vendor_tray_meal/pesto_pizza
+	name = "\improper Main Course - Type M: Pesto Pizza"
+	desc = "A rectangular pizza with a suspiciously bright green pesto in place of the standard tomato sauce."
+	icon_state = "foodtray_pestopizza"
+	tastes = list("tomato?" = 2, "cheese?" = 2, "herbs" = 1, "laziness" = 1)
+	foodtypes = GRAIN | DAIRY | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/baked_rice
+	name = "\improper Main Course - Type M: Baked Rice and Grilled Cheese"
+	desc = "Some sub-par looking fleet style rice, with a generous hunk of quite literally grilled cheese."
+	icon_state = "foodtray_ricengrilledcheese"
+	tastes = list("rice" = 2, "peppers" = 2, "charred cheese" = 2, "laziness" = 1)
+	foodtypes = GRAIN | DAIRY | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/fueljack
+	name = "\improper Main Course - Type M: Fueljack's Tray"
+	desc = "A flat chunk of fueljack's lunch, seemingly missing most of the usual variety in ingredients."
+	icon_state = "foodtray_fueljacksmeal"
+	tastes = list("potato" = 2, "cabbage" = 2, "cheese?" = 2, "laziness" = 1)
+	foodtypes = DAIRY | VEGETABLES
+
+/obj/item/food/vendor_tray_meal/moonfish_nizaya
+	name = "\improper Tiziria Imports: Moonfish and Nizaya"
+	desc = "An almost synthetic looking cut of moonfish, paired with an equal helping of nizaya pasta."
+	icon_state = "foodtray_moonfishnizaya"
+	tastes = list("fish?" = 2, "cheap noodles" = 2, "laziness" = 1)
+	foodtypes = VEGETABLES | NUTS | SEAFOOD
+
+/obj/item/food/vendor_tray_meal/emperor_roll
+	name = "\improper Tiziria Imports: Emperor Roll"
+	desc = "A pretty sad looking emperor roll, if you can even call it that, it seems caviar wasn't in the budget."
```

Comment:
```suggestion
	desc = "A pretty sad looking emperor roll, if you can even call it that; it seems caviar wasn't in the budget."
```
Rather than explain in my own bullshit way I looked up 'when to use a semicolon' on google lmao:
`Use a semicolon to join two related independent clauses in place of a comma and a coordinating conjunction (and, but, or, nor, for, so, yet). Make sure when you use the semicolon that the connection between the two independent clauses is clear without the coordinating conjunction.`

Because the latter half is adding onto the former half (sad emperor roll - caviar wasn't used) we can get the effect of one sentence without it feeling like a run-on.
Another quote from the google search that could explain this - `"At a comma, stop a little; at a semicolon, somewhat more." -Richard Hodges, The English Primrose (1644)`